test/test_unary_ufuncs.py

```diff
@@ -490,6 +490,55 @@ def test_sqrt_complex_edge_values(self, device, dtype):
         x = torch.tensor(-1.0000e+20 - 4988429.2000j, dtype=dtype, device=device)
         self.compare_with_numpy(torch.sqrt, np.sqrt, x)
 
+    @dtypes(torch.cfloat, torch.cdouble)
+    def test_abs_complex_edge_values(self, device, dtype):
+        values = [
```

Comment:
This is a great set of values. Instead of just defining it here I think we should define it here as `hard_complex_values.` Another list can be `hard_complex_double_values` for the additional double values. This can go where we define the other value lists here:

https://github.com/pytorch/pytorch/blob/ef41709e8ad6acacc7efc87298a7c069660bde56/test/test_unary_ufuncs.py#L55

One way to construct these would be to create tuples of values and then turn them into complex numbers programmatically. See here for an example:

https://github.com/pytorch/pytorch/blob/ef41709e8ad6acacc7efc87298a7c069660bde56/test/test_unary_ufuncs.py#L105

That would save you from having to write the cross-product of values manually.